CMakeLists.txt

```diff
@@ -303,6 +303,7 @@ endif ()
 
 
 if (MSVC)
+    add_definitions (-DNOMINMAX)
```

Comment:
OK. I got errors from: oiio/src/include/OpenImageIO/fmath.h
Due to all use of std::numeric_limits<float>::min() and std::numeric_limits<float>::max()
The error is preceded by : warning C4003: nombre de paramètres réels insuffisants pour la macro 'max'
It caused by the windows.h include which define min and max macro : http://stackoverflow.com/questions/4913922/possible-problems-with-nominmax-on-visual-c
I saw a workaround here:  oiio/src/include/OpenImageIO/platform.h
but it seem to be not enough...